src/views/Transformer.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <main>
+    <ViewHeader :title="transformer.name" :backLink="{ link: { name: 'transformers' } }"/>
+
+    <ListSection class="preview-container" :list="transformerList"/>
+
+    <div class="row">
+      <ListSection title="Attributes" :list="transformerAttributesList"/>
```

Comment:
![image](https://user-images.githubusercontent.com/1167788/60663869-b5767880-9e60-11e9-9419-521643202cbf.png)
The list ignores line breaks. (The API return `\n` for line breaks, but here they are omitted somehow.)
It is not a big problem, but might be nice for longer codes snippets.
![image](https://user-images.githubusercontent.com/1167788/60663949-efe01580-9e60-11e9-906f-c213521102f0.png)

Reply:
The linebreaks are transmitted correctly, but in the last step, the text is written inside of <p> elements (in Li.vue) , which seemingly are unable to display the linebreaks correctly. 
the browser console still knows, where the breaks are.

Reply:
![image](https://user-images.githubusercontent.com/1167788/60668220-bf51a900-9e6b-11e9-845e-3e024a13ea02.png)
In css, try adding the rule `white-space: pre;` :D

Reply:
Thanks for mentioning this, it is now included.